content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -363,6 +363,26 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
+#### Write Access for Endpoints
+
```

Comment:
I would place this content near the end of the doc, well after the "User-facing roles" section where the edit and admin roles are described (maybe just before the "Upgrading from ABAC" section?)

also, in the description of the `admin` and `edit` roles in the "User-facing roles" section, update the description to indicate write permissions to endpoints are not included in clusters created on 1.22+ and link to this section for details.

Reply:
Updated, thanks!